services/order.py

```diff
@@ -0,0 +1,40 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession, User
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    user = User.objects.get(username=username)

```

Comment:
you should use `get_user_model()` to get user and this should also be under atomic decorator

Reply:
yeah, my mistake there is no need for transaction in read operation here :blush: